### ISSUE TITLE
A11y_SQLAzureDataStudio_ExportBacpac_SummaryScreen_ScreenReader: Voice over annouces "empty group" while announcing the cell value. 

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#9262]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11y_SQLAzureDataStudioBenchmark;#A11yMAS;#A11ySev2;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#DesktopApp;#MacOs;#MAS1.3.1;#VoiceOver;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.15.0-insider
Commit: 3a8b74a
Date: 2020-02-11T05:49:41.181Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.3.0
Mac OS:10.15.3

**Additional Details:**
Screen reader: Voiceover
Mac OS:10.15.3
MAS References:MAS1.3.1

**Repro Steps:**
1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Extensions from left menu.
4. Search & install Dacpac extension.
5. Navigate to Connections from left menu.
6. Expand the server connected in step 2.
7. Right click an on prem database and select Data-tier application wizard from Context menu.
8. Select forth radio button.
9. Navigate till next button & hit enter.
10. Provide all the details & hit enter.
11. Launch Voiceover and navigate using control+option(left/right) arrow key.

**Actual:**
On navigating using the keys control+option (left/right) arrow key to a cell in the table it announces the name the column association and empty group.

**Same issue in the following page:**
1.Import Bacpac-Summary Screen

**Expected:**
The empty group should be not announced when navigating to each cell fo the table.

**User Impact:**
Screen readers users will receive incorrect information.

**Attachment link for Reference :**
<img width="2545" alt="Screenshot 2020-02-21 at 1 13 15 PM" src="https://user-images.githubusercontent.com/24580570/75026253-a4977780-54c2-11ea-9650-7887ff3b82c4.png">

**Comment by alanrenmsft** [comment]
need to figure why it is announcing empty group

**Comment by kisantia** [comment]
Not able to repro
![image](https://user-images.githubusercontent.com/31145923/76812594-96333780-67b2-11ea-9a34-d6deef263e43.png)

**Comment by customcommander** [comment]
@kisantia Were you on macOS 10.14 when you couldn't repro? (Currently facing a similar situation and we're not sure why it's happening.)

**Comment by kisantia** [comment]
I'm currently on macOS 10.15.7 and still can't repro it

**Comment by BhanuPrakashGit** [comment]
Verified the bug and does not repro.
**GithubTags:**#Closed;
PFA for reference
[9262_Closed.zip](https://github.com/microsoft/azuredatastudio/files/10732252/9262_Closed.zip)

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-MAC-Feb2020;
